vendor/github.com/petar/GoLLRB/.gitignore

```diff
@@ -0,0 +1,23 @@
+syntax:glob
```

Comment:
vendor/github.com/google/btree/btree_mem.go imports that and  glide either didn't detect that or was smarter.